docs/changelog/changelog_de.md

```diff
@@ -1,5 +1,11 @@
 # Changelog plentyshopPWA
 
+## v1.6.1
+
+- Neue Karussellkomponente über die Swipper-Bibliothek hinzugefügt
+- Neues useCarousel Composable mit Karusselllogik erstellt
+- Minimierte und bereinigte Swiper-CSS-Dateien zur Geschwindigkeitsoptimierung
+
```

Comment:
**todo**: Please remove the German changelog again. We've found it's ultimately less confusing to translate the English one in one go when we do the release.